src/notifications.cpp

```diff
@@ -1199,3 +1199,8 @@ void addNotification(const WZ_Notification& notification, const WZ_Notification_
 	// Add the notification to the notification system's queue
 	notificationQueue.push_back(std::unique_ptr<WZ_Queued_Notification>(new WZ_Queued_Notification(notification, WZ_Notification_Status(realTime), trigger)));
 }
+
+bool hasActiveNotificaton()
+{
+	return currentInGameNotification != nullptr;
```

Comment:
We need a better way to identify specific notifications, as there may be multiple consumers of the Notification system. (What you really want is "is there an active _`voting`_ notification".) I'll work up a PR with some improvements.